### PR TITLE
Implement AMD leaf 0x8000_001A (Performance Optimization Info)

### PR DESCRIPTION
**Pull request by Umio-Yasuno**
Related: #58

### CODE REVIEW

File: src/extended.rs [hunk 1]
```diff
@@ -1583,3 +1583,57 @@ bitflags! {
         const TLB_CTL = 1 << 24;
     }
 }
+
+/// Performance Optimization Identifier (LEAF=0x8000_001A).
+///
+/// # Note
```
Comment (gz) [inline src/extended.rs:1589]: Maybe drop empty `# Note` here?

**Comment by gz** [comment]
Thanks for the contribution! This looks good just have to fix the `# Note` thing.

**Comment by gz** [comment]
Do you mind printing the leaf in the binary too?

e.g., here:
https://github.com/gz/rust-cpuid/blob/master/src/bin/cpuid.rs#L1442

**Comment by Umio-Yasuno** [comment]
Sorry, I'm not familiar with conflict resolution.

**Comment by gz** [comment]
No worries that was my fault -- just pushed it and released your PRs as crate version 10.5 on crates.io, thanks a lot again really appreciate these additions.

**Comment by Umio-Yasuno** [comment]
Thanks @gz.

**Pull request merged**